change/@ni-nimble-components-73767256-fc9b-453d-b09b-6f15dc7d5f72.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

If you make the change type `none`, then you won't get a new npm package published, which isn't what you want. We generally consider styling changes `patch`, but that's definitely the case here since we make all changes to incubating components patches.

```suggestion
  "type": "patch",
```